app/dataBuilder/car.py

```diff
@@ -0,0 +1,33 @@
+import app.dataBuilder.generator.generate as gen
+
+
+class Car:
+    def __init__(self):
```

Comment:
gen.generatecar is running 5 times instead of one, Could be fixed with
`car_dict = gen.generate_car()
self.brand_name = card_dict['brand_name']`
and so on